addon/components/a11y-landmark.js

```diff
@@ -60,5 +87,22 @@ export default Ember.Component.extend({
         }
     }),
 
+    _validateProperties() {
+        this._validateTagName(this.tagName);
+        this._validateLandmarkRole(this.landmarkRole);
+    },
 
-});
\ No newline at end of file
+    _validateTagName(tagName) {
```

Comment:
can probably DRY this up. something like:
```js
_validate(attribute, list) {
  if (list.indexOf(attribute) === -1) { return; }
  const validValues = list.join(', ');
  Ember.assert(`Invalid tagName "${attribute}". Must be one of ${validValues}.`);
} 
```

If you want, you can make the lists a string in the format of validValues since you're going to join them every time anyway. Not a significant performance improvement, but you can see the effects: https://jsperf.com/stringarrayindexof